tuist/tuist
# Appclip support

Resolves https://github.com/tuist/tuist/issues/1815

Add support for AppClips 

### Short description 

Adds support for AppClips product.

### Solution 

Add support for AppClips products with appropriate dependency rules.

### Implementation

- [x] Add product type
- [x] Add fixture & acceptance test
- [x] Update graph / link rules
- [x] Add embed AppClips build phase
- [x] Add AppClips build settings
- [x] Add linter rules for bundle IDs
- [x] Add linter rules for valid product types

### Test Plan

- Run `tuist generate` within `fixtures/ios_app_with_appclips`
- verify the generated project contains an AppClips target
- verify the generated project contains main app target with Embed App Clips build phase
- Compare against a manually created project with an AppClips
